src/modules/moon-pay/components/MoonPaySellComponent.vue

```diff
@@ -421,6 +423,7 @@ export default {
     this.locGasPrice = this.gasPriceByType(this.gasPriceType);
   },
   methods: {
+    ...mapActions('external', ['setCoinGeckoTokens']),
```

Comment:
Where is this being used?

Reply:
It is being called in the walletHandler mixin, when I remove it I get an error.
![image](https://user-images.githubusercontent.com/36307949/182914797-2f10680e-acf4-4c56-973f-c44ec8d06799.png)